scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -102,9 +92,9 @@
                      'Optional (defaults to "docker_api"). Determine which method is used to gather logs from the '
                      'local containers. If "docker_api", then this agent will use the docker API to contact the local '
                      'containers and pull logs from them.  If "syslog", then this agent expects the other containers '
-                     'to push logs to this one using the syslog Docker log plugin.  Currently, "syslog" is the '
-                     'preferred method due to bugs/issues found with the docker API.  It is not the default to protect '
-                     'legacy behavior.\n',
+                     'to push logs to this one using the Docker syslog logging driver.  Currently, "syslog" is the '
+                     'preferred method due to bugs/issues found with the docker API (To protect legacy behavior, '
+                     'the default method is "docker_api").',
```

Comment:
I reworded this a bit to:
a) Use the official docker name for their syslog plugin
b) Slightly sharpen the English in the last sentence.